src/dbt_client/dbtIntegration.ts

```diff
@@ -346,6 +346,7 @@ export interface DBTProjectIntegration extends Disposable {
   getCatalog(): Promise<Catalog>;
   getDebounceForRebuildManifest(): number;
   getBulkSchema(nodes: DBTNode[]): Promise<Record<string, DBColumn[]>>;
+  findPackageVersion(packageName: string): string | undefined;
```

Comment:
```suggestion
  findPackageVersion(packageName: string): string;
```
What will you do when it doesn't return string?

Reply:
Current requirement does not expect the version to be always available. If version is not returned, we can assume that the package is not installed and wont use it. I have updated it to return string or throw error, so the caller will handle this as per the need